threadsd/main.go

```diff
@@ -73,23 +79,40 @@ func main() {
 	log.Debugf("connGracePeriod: %v", *connGracePeriod)
 	log.Debugf("keepAliveInterval: %v", *keepAliveInterval)
 	log.Debugf("enableNetPubsub: %v", *enableNetPubsub)
+	if *mongoUri != "" {
+		log.Debugf("mongoUri: %v", *mongoUri)
```

Comment:
This might be a bit of a security issue since the URI contains the password and that can end up living in some log file or external logging systems.

Reply:
Switched to using the `url` package to parse the URL and then display a redacted string like ` mongoUri: mongodb://boom:xxxxx@127.0.0.1:27017`